### ISSUE TITLE
Site Assembler - Large preview with patterns accessibility

### ISSUE THREAD

**Issue by miksansegundo** [Automattic/wp-calypso#74834]
### What
Allow getting into the large preview iframe with the keyboard to scroll up/down and/or move the focus through the patterns and their action toolbar

### Why
Currently the user does not have a way to move the focus inside the large preview iframe using the keyboard to explore the patterns added by the user

### How
Investigate how the editor does it. See also https://wpcalypso.wordpress.com/devdocs/docs/accessibility.md#utilities-in-calypso

**Comment by miksansegundo** [comment]
The keyboard accessibility is working, users can move the focus to the patterns.

https://github.com/Automattic/wp-calypso/assets/1881481/696ad271-17d1-4ca0-9aec-1187f73b0b18

We'll have to open it only if we merge the [single iframe solution](https://github.com/Automattic/wp-calypso/issues/73757)
